pkg/services/govmomi/ipam/status_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	apitypes "k8s.io/apimachinery/pkg/types"
-	ipamv1a1 "sigs.k8s.io/cluster-api/exp/ipam/api/v1alpha1"
+	ipamv1b1 "sigs.k8s.io/cluster-api/exp/ipam/api/v1beta1"
```

Comment:
please rename this to `ipamv1`, maybe worth enforcing this via `.golangci.yml` like we already do for other apis?

Reply:
Yeah, sure. Added a rule to .golangci.yml as well.